mule-extensions-api-persistence/src/test/resources/extension/list-of-serialized-extension-model.json

```diff
@@ -53,6 +53,190 @@
                 "order": 0
               }
             }
+          },
+          {
+            "type": {
+              "format": "java",
+              "type": "Object",
+              "annotations": {
+                "typeId": "org.mule.runtime.extension.api.persistence.model.ComplexFieldsType",
+                "classInformation": {
+                  "hasDefaultConstructor": true,
+                  "isInterface": false,
+                  "isInstantiable": true,
+                  "isAbstract": false,
+                  "isFinal": false,
+                  "implementedInterfaces": [],
+                  "parent": "",
+                  "genericTypes": []
+                },
+                "org.mule.runtime.extension.api.introspection.declaration.type.annotation.TypeAliasAnnotation": {
+                  "value": "complex-alias"
+                }
+              },
+              "fields": [
+                {
+                  "key": {
+                    "name": "extensibleTypeList"
+                  },
+                  "model": {
+                    "required": "true",
+                    "type": "Array",
+                    "annotations": {
+                      "typeId": "java.util.List"
+                    },
+                    "item": {
+                      "type": "Object",
+                      "annotations": {
+                        "typeId": "org.mule.runtime.extension.api.persistence.model.ExtensibleType",
+                        "classInformation": {
+                          "hasDefaultConstructor": true,
+                          "isInterface": false,
+                          "isInstantiable": true,
+                          "isAbstract": false,
+                          "isFinal": false,
+                          "implementedInterfaces": [],
+                          "parent": "",
+                          "genericTypes": []
+                        },
+                        "org.mule.runtime.extension.api.introspection.declaration.type.annotation.ExtensibleTypeAnnotation": {},
```

Comment:
is the first annotation

`"typeId": "org.mule.runtime.extension.api.persistence.model.ExtensibleType",`